aten/src/ATen/gen.py

```diff
@@ -192,6 +200,10 @@ def check_all_files_written(self):
     ('QInt8', 'qint8', 'Long', False),
 ]
 
+quantized_scalar_types = [
+    ('QInt8', 'qint8', 'Long', False),
```

Comment:
do we actually need to defined Accreal and is_floating_type here?  What they should be isn't obvious and I'm not sure they actually matter.

You can see with `Bool` above the accreal is `BoolAccrealNotDefined` which just checks we don't generate something silly.  And I'm not convinced is_floating_type is actually used anywhere, you could try doing something similar here.

Reply:
?? is this comment for another PR?

Reply:
is_floating_point is still False right?